spec/models/document_spec.rb

```diff
@@ -7,77 +7,37 @@
       language: 'en',
       path: 'http://www.agency.gov/page1.html',
       title: 'My Title',
-      created: DateTime.now,
-      changed: DateTime.now,
+      created: DateTime.new(2020, 1, 1),
+      changed: DateTime.new(2020, 1, 2),
       description: 'My Description',
       content: 'some content',
       promote: true,
-      tags: 'this,that'
+      tags: 'this,that',
+      click_count: 5
     }
   end
 
-  before(:all) do
-    handle = 'test_index'
-    Elasticsearch::Persistence.client.indices.delete(
-      index: [Document.index_namespace(handle), '*'].join('-')
-    )
-    es_documents_index_name = [Document.index_namespace(handle), 'v1'].join('-')
-    Document.create_index!(index: es_documents_index_name)
-    Elasticsearch::Persistence.client.indices.put_alias index: es_documents_index_name,
-                                                        name: Document.index_namespace(handle)
-    Document.index_name = Document.index_namespace(handle)
-  end
-
-  after(:all) do
-    Elasticsearch::Persistence.client.indices.delete(
-      index: [Document.index_namespace('test_index'), '*'].join('-')
-    )
```

Comment:
These `before`/`after` blocks were there to support the `.create` specs, which have now been moved to the Serde specs.